packages/dataviews/src/stories/dataform.story.tsx

```diff
@@ -872,279 +706,503 @@ const ValidationComponent = ( {
+				label: 'telephone',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customTelephoneRule ),
+				},
 			},
-		},
-		{
-			id: 'color',
-			type: 'color',
-			label: 'Color',
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
-				custom: maybeCustomRule( customColorRule ),
+			{
+				id: 'url',
+				type: 'url',
+				label: 'URL',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customUrlRule ),
+				},
 			},
-		},
-		{
-			id: 'integer',
-			type: 'integer',
-			label: 'Integer',
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
-				custom: maybeCustomRule( customIntegerRule ),
+			{
+				id: 'color',
+				type: 'color',
+				label: 'Color',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customColorRule ),
+				},
 			},
-		},
-		{
-			id: 'number',
-			type: 'number',
-			label: 'Number',
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
-				custom: maybeCustomRule( customNumberRule ),
+			{
+				id: 'integer',
+				type: 'integer',
+				label: 'Integer',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customIntegerRule ),
+				},
 			},
-		},
-		{
-			id: 'boolean',
-			type: 'boolean',
-			label: 'Boolean',
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
-				custom: maybeCustomRule( customBooleanRule ),
+			{
+				id: 'number',
+				type: 'number',
+				label: 'Number',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customNumberRule ),
+				},
 			},
-		},
-		{
-			id: 'countries',
-			label: 'Countries Visited',
-			type: 'array' as const,
-			placeholder: 'Select countries',
-			description: 'Countries you have visited',
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
+			{
+				id: 'boolean',
+				type: 'boolean',
+				label: 'Boolean',
+				isValid: {
+					required,
+					elements: elements !== 'none' ? true : false,
+					custom: maybeCustomRule( customBooleanRule ),
+				},
 			},
-			elements:
-				elements === 'async'
-					? undefined
-					: [
-							{ value: 'us', label: 'United States' },
-							{ value: 'ca', label: 'Canada' },
-							{ value: 'uk', label: 'United Kingdom' },
-							{ value: 'fr', label: 'France' },
-							{ value: 'de', label: 'Germany' },
-							{ value: 'jp', label: 'Japan' },
-							{ value: 'au', label: 'Australia' },
-					  ],
-			getElements:
-				elements === 'async' ? getElements( 'countries' ) : undefined,
-		},
-		{
-			id: 'customEdit',
-			label: 'Custom Control',
-			Edit: CustomEditControl,
-			isValid: {
-				required,
-				elements: elements !== 'none' ? true : false,
+			{
+				id: 'array',
```

Comment:
changed `countries` to `array` because any other field type just uses the type as the id/label.